.travis.yml

```diff
@@ -16,18 +16,33 @@ env:
     - secure: "bWJ1eCA19Qt8dHarlUC/5SfgkOk08VdASljeC7NVy7jAzNnhZna6My39BkBWHUxDGGN6PxeaNzb8COMo94dcfk5OoIQ8ZB8CXwtg/o5Unpi8S2uNC1+H6SvR06xYXPCNozqPwBO151HAK+8IhEpJidQ0Mj1qjrYC3BiWePrXwgs="
 # Setup anaconda for easily running scipy on Travis; see https://gist.github.com/dan-blanchard/7045057
```

Comment:
I guess you can remove the `dan-blanchard` reference, since now you're using `conda`'s recommended code?